requirements.txt

```diff
@@ -1,2 +1,6 @@
-pytest==6.2.5
-flake8==4.0.1
\ No newline at end of file
+flake8==5.0.4
```

Comment:
Do not change the `requirements.txt`, this file should be absent in the pull request.